.github/workflows/package-build.yaml

```
@@ -0,0 +1,57 @@
+name: Mautic Language Packer
+
+run-name: Action performed by ${{ github.actor }}
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  TRANSIFEX_API_TOKEN: ${{ secrets.TRANSIFEX_API_TOKEN }}
+  TRANSIFEX_ORGANISATION: ${{ secrets.TRANSIFEX_ORGANISATION }}
+  TRANSIFEX_PROJECT: ${{ secrets.TRANSIFEX_PROJECT }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: php:8.1
+    steps:
+      - name: Install PHP zip extension
+        run: apt-get update && apt-get install -y libzip-dev && docker-php-ext-install zip
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install composer
+        run: |
+          curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+      - name: Install dependencies
+        run: |
+          apt-get update && apt-get install -y git jq ssh unzip zip
+          mkdir -p ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
+          composer install
+
+      - name: Build packages
+        run: |
+          bin/console m:l:p
```

```suggestion
          bin/console mautic:language:packer
```
Using command shortcuts in scripts may lead to issues in the future.